packages/blockchain-wallet-v4-frontend/src/scenes/TaxCenter/selectors.ts

```diff
@@ -51,9 +51,15 @@ export const getAddressesAndXpubs = createDeepEqualSelector(
       return {
         bchAddresses: filter((x: string) => not(startsWith('xpub', x)), bchContext),
         bchXPubs: filter((x: string) => startsWith('xpub', x), bchContext),
-        btcBech32ImportedAddresses: btcContext.addresses,
+        btcBech32ImportedAddresses: filter(
+          (x: string) => startsWith('bc1', x),
```

Comment:
you could put this duplicated hardcoded `bc1` in a single constant and re-used in line 60. Just a comment, not a blocker point